### ISSUE TITLE
log.Fatal instead of panics

### ISSUE THREAD

**Issue by hypersleep** [nanopack/yoke#7]
There are so many panic messages when i try to play with yoke.

For example:

```
bash-4.2$ yoke /etc/yoke/yoke.ini
2015-11-18 16:32:13 INFO  Creating scribble database at 'status'...
2015-11-18 16:32:13 INFO  database has already been created... skipping.
2015-11-18 16:32:13 INFO  going to start
2015-11-18 16:32:13 INFO  starting
2015-11-18 16:32:13 INFO  [action] starting db
panic: exec: "postgres": executable file not found in $PATH

goroutine 1 [running]:
main.main()
	/root/yoke/main.go:92 +0xfef

goroutine 5 [syscall]:
os/signal.loop()
	/usr/lib/golang/src/os/signal/signal_unix.go:21 +0x1f
created by os/signal.init·1
	/usr/lib/golang/src/os/signal/signal_unix.go:27 +0x35

goroutine 7 [runnable]:
net/rpc.(*Server).Accept(0xc2080d8c80, 0x7f1809d93628, 0xc2080380e0)
	/usr/lib/golang/src/net/rpc/server.go:615
created by github.com/nanopack/yoke/state.(*state).ExposeRPCEndpoint
	/root/go/src/github.com/nanopack/yoke/state/rpc.go:59 +0x242

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/lib/golang/src/runtime/asm_amd64.s:2232 +0x1

goroutine 8 [runnable]:
github.com/nanopack/yoke/monitor.func·001()
	/root/go/src/github.com/nanopack/yoke/monitor/action.go:58
created by github.com/nanopack/yoke/monitor.NewPrefix
	/root/go/src/github.com/nanopack/yoke/monitor/action.go:62 +0x27a

goroutine 9 [runnable]:
github.com/nanopack/yoke/monitor.func·001()
	/root/go/src/github.com/nanopack/yoke/monitor/action.go:58
created by github.com/nanopack/yoke/monitor.NewPrefix
	/root/go/src/github.com/nanopack/yoke/monitor/action.go:62 +0x27a
```

Maybe use a log.Fatal instead of panics to show only useful information? Obviously in this case program should only to show: ` exec: "postgres": executable file not found in $PATH` without any stack traces and exited with error status code.

**Comment by hypersleep** [comment]
I've already started to resolve this issue. I'll try to send a PR when it's ready.

**Comment by sdomino** [comment]
resolved in #9
